### DISCUSSION TITLE
Where is the new dpmpp_2s_ancestral_cfg_pp sampler?

### DISCUSSION THREAD

**Discussion by aikonaut** [Comfy-Org/ComfyUI#4911] [General]
I can't find it anywhere.

**Comment by LukeG89** [comment]
You should find it in KSampler

![Ksampler](https://github.com/user-attachments/assets/cdc41489-7b71-46a6-8835-8141020a74c3)

**Comment by ltdrdata** [comment]
Update ComfyUI to latest version

**Reply by aikonaut** [reply]
Thanks for the quick response. Unfortunately, I still can't see the sampler. 
Of course, before posting my question, I updated ComfyUI - by downloading it from the releases page https://github.com/comfyanonymous/ComfyUI/releases. It's a 1,582,606 KB file. I extracted it contents over the existing older version of portable ComfyUI installed on my D: drive, non-system, it's my own computer, I have all sysadmin privileges. I also did a clean install into a different folder. Nothing new showed up. I also did an upgrade from the Manager. 
With all due respect to the huge contribution of the developers and their brilliant minds and work, I assume the problem is that the latest version number is not specified in the file name, so it is not clear if it is the latest version or not, all the files have the same name - ComfyUI_windows_portable_nvidia.7z. It is also not clear if the versions listed in ComfyUI Settings ( 1.2.47 ) and ComfyUI Manager ( ComfyUI: 2652[0c7c98](2024-09-05). Manager: V2.50.3 ) is the latest. The version number on the comfyanonymous/ComfyUI page is v0.2.2. These numbers are not comparable. 
I repeated the installation procedure right now - nothing has changed. Maybe I'm doing something wrong )))

**Reply by aikonaut** [reply]
p.s. I also want to add that I care about this question for a reason, as I found the dpmpp_2s_ancestral sampler is very successful in some cases - slow but high quality, so I was interested to see what has changed in the new version of _cfg_pp.

**Reply by ltdrdata** [reply]
Your ComfyUI is latest stable release but not latest version.

goto ComfyUI dir in cmd
And execute 

```
git switch master
git pull
```

That should not be failed.
